### PR TITLE
add map option in autoprefixer task so that sourceMappingURL is not removed

### LINKED ISSUE #455: Sass sourcemap not displaying
With a fresh install, running grunt serve, inspecting my page elements references main.css, rather than any of my scss partials. Although a sourcemap is being rendered each time in .tmp directory, it fails to show up in Web Inspector. :(

### PR DESCRIPTION
**Pull request by gaboesquivel**
add map option in autoprefixer task so that sourceMappingURL is not removed

fixes #455

### CODE REVIEW

File: app/templates/Gruntfile.js [hunk 1]
```diff
@@ -222,7 +222,11 @@ module.exports = function (grunt) {
     // Add vendor prefixed styles
     autoprefixer: {
       options: {
-        browsers: ['> 1%', 'last 2 versions', 'Firefox ESR', 'Opera 12.1']
+        browsers: ['> 1%', 'last 2 versions', 'Firefox ESR', 'Opera 12.1']<% if (includeSass) { %>,
+        map: {
+          prev:'.tmp/styles/'
```
Comment (sindresorhus) [inline app/templates/Gruntfile.js]: `prev: '.tmp/styles/'`

Comment (gaboesquivel) [inline app/templates/Gruntfile.js]: squashed

**Comment by sindresorhus** [comment]
Have you read the docs? `prev` seems to expect a sourcemap string/object: https://github.com/postcss/autoprefixer-core#css-processing

**Comment by gaboesquivel** [comment]
grunt-autoprefixer takes care of it, see getPrevMap...  grunt-autoprefixer expects a path to a directory where a previous sourcemap is

**Pull request merged**